recipes/pyramid_mako/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pyramid_mako" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0066c863441f1c3ddea60cee1ccc50d00a91a317a8052ca44131da1a12a840e2
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+    - setuptools
+  run:
+    - python >=2.7
+    - mako >=1.1.0
+    - pyramid
+
+test:
+  imports:
+    - pyramid_mako
+  # requires:
+  #   - webtest >=1.3.1
+
+about:
+  home: https://trypyramid.com
+  license: BSD-derived
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: The Pyramid Web Framework, a Pylons project
+  doc_url: https://docs.pylonsproject.org/projects/pyramid_mako/en/latest/
+  dev_url: https://github.com/Pylons/pyramid_mako
+
+extra:
+  recipe-maintainers:
+    - ChrisBarker-NOAA
+    - JamesMakela-NOAA
```

Comment:
@JamesMakela-NOAA Are you happy to be listed as a recipe maintainer here?

Reply:
> @JamesMakela-NOAA Are you happy to be listed as a recipe maintainer here?

Yes, that would be fine.